menas/src/main/scala/za/co/absa/enceladus/menas/controllers/ConfigurationController.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.controllers
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.web.bind.annotation.{GetMapping, RequestMapping, RestController}
+
+@RestController
+@RequestMapping(Array("/api/configuration"))
+class ConfigurationController extends BaseController {
+  @Value("${za.co.absa.enceladus.menas.environment}")
```

Comment:
Is this a required parameter?

Reply:
Yes, build should fail without it.

Reply:
This seems little constricting. (In conjunction with my previous comment, if `properties` file is the proper location for this info.)